src/main/java/core/basesyntax/service/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operations.Operation;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<FruitTransaction> read(String filePath) {
```

Comment:
This method should only return a list of read lines. The parsing of these strings in the transaction should be in another service. Otherwise it violates the principles of solid